.changelog/2036.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+api-gateway: Add GatewayClassConfig CRD to allow for configuration of GatewayClass.
```

Comment:
Obviously not part of this PR, but I imagine we'll also need to update our install docs to reflect this improvement.

Reply:
Definitely! We'll have a whole rewrite of install and upgrade docs.